src/services/spotlight/service/util.mjs

```diff
@@ -1,11 +1,10 @@
 /* eslint-disable no-await-in-loop */
-import * as path from 'path';
-import { fileURLToPath } from 'url';
-
 import * as _ from 'lamb';
+import * as path from 'path';
 import { fetch } from 'undici';
+import { fileURLToPath } from 'url';
```

Comment:
we usually separate node imports from npm imports, so I guess the change in this file should be reverted